test/models/paye/PayeAtsDataTest.scala

```diff
@@ -36,19 +36,45 @@ import config.ApplicationConfig
 import models.LiabilityKey._
 import models.RateKey._
 import models._
-import org.scalatestplus.play.OneAppPerSuite
+import org.mockito.Mockito.when
+import org.scalatest.BeforeAndAfterEach
+import org.scalatest.mockito.MockitoSugar
 import services.GoodsAndServices
 import services.GoodsAndServices._
 import uk.gov.hmrc.play.test.UnitSpec
-import utils.{BaseSpec, PayeAtsDataUtil, TestConstants}
+import utils.{PayeAtsDataUtil, TestConstants}
 
-class PayeAtsDataTest extends BaseSpec {
+class PayeAtsDataTest extends UnitSpec with MockitoSugar with BeforeAndAfterEach {
 
   val atsData: PayeAtsData = PayeAtsDataUtil.atsData
   val nino: String = TestConstants.testNino
   val taxYear = "2018"
+
+  val mockAppConfig = mock[ApplicationConfig]
+
+  when(mockAppConfig.writEnabled).thenReturn(true)
+
   lazy val transformedData: PayeAtsMiddleTier =
-    atsData.transformToPayeMiddleTier(applicationConfig, nino, taxYear.toInt)
+    atsData.transformToPayeMiddleTier(mockAppConfig, nino, taxYear.toInt)
+
+  when(mockAppConfig.governmentSpend(taxYear.toInt + 1)).thenReturn(
```

Comment:
I think this was better before when you used the real `ApplicationConfig` I can have a look with you on changing this